### PR TITLE
Kraken-mpa-report will now use element_identifier instead of generic sample_i for sample names.

### PR DESCRIPTION
**Pull request by blankenberg**

### CODE REVIEW

**Comment by bgruening** [comment]
@blankenberg one test is failing.

**Comment by bgruening** [comment]
@blankenberg can you add a lines_diff to fix this test.
```
-#Sample ID	sample_0	sample_1
+#Sample ID	kraken_mpa_report_input1.tab	kraken_mpa_report_input2.tab
```

**Comment by blankenberg** [comment]
@bgruening thanks for the review!

**Comment by bgruening** [comment]
Thanks!

File: tool_collections/kraken/kraken_report/kraken-mpa-report.xml [hunk 1]
```diff
@@ -10,14 +10,24 @@
     <command>
 <![CDATA[
 
+#set $names = []
+
 #for $input_count, $input_classification in enumerate( $classification ):
-    ln -s "${input_classification}" "sample_${input_count}" &&
+    #set $name_base = str( getattr( $input_classification, 'element_identifier', 'sample' ) ).replace( "/", '-' ).replace( "\t", "-" )
```
Comment (jmchilton) [inline tool_collections/kraken/kraken_report/kraken-mpa-report.xml]: Element identifier is populated for all dataset wrappers - is this to explicitly enable like really old Galaxies or is there some edge case I am missing?

Comment (blankenberg) [inline tool_collections/kraken/kraken_report/kraken-mpa-report.xml]: Correct.

**Pull request merged**